float32 temp_aV
float32 temp_urgency
float32 adsign
float32 hsign
